.github/workflows/checks.yml

```diff
@@ -52,3 +52,30 @@ jobs:
             - name: Push changes
               run: |
                   git push
+
+    js_checks:
+        name: JS Checks
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '16'
+            - name: Install dependencies
+              run: yarn
```

Comment:
You mentioned this step takes a while — what if we ran this locally with husky instead of using GH actions? https://github.com/typicode/husky#readme

Then, presumably, the user making a commit would have the local dependencies installed already.

Reply:
I'm not super-happy with husky playing with my local commit history but I'll leave that aside here.

As you say though, it would mean the tests will run much faster on the machine of the person making the commit vs. having this happen in a GH action where all the dependencies need to be installed.

@smallbrownbike - any thoughts on this from your perspective?

I'll give this 24 hours and then make a call.

Reply:
Since some of the team makes changes directly on github.com, I would say it makes the most sense to keep this a GitHub action. 😄